.github/workflows/gallery.yml

```diff
@@ -22,9 +22,7 @@ jobs:
       - name: Build Gallery 🔧
         run: docker run -v "$(pwd):/work" thumbsupgallery/thumbsup /bin/sh -c "cd /work/ && thumbsup --config config.json"
       - name: Deploy to Github Pages 🚀
-        uses: JamesIves/github-pages-deploy-action@4
+        uses: actions/deploy-pages@0243b6c10d06cb8e95ed8ee471231877621202c0
```

Comment:
using `actions/deploy-pages@v1` would be a better idea, ref: https://github.com/marketplace/actions/deploy-github-pages-site

Reply:
I let you decide what you want for your project but in term of security, it's better to use hash. :smile: 


> Consider this threat scenario: you are using a third-party action that runs a linter on your code to check for formatting issues. Rather than install, configure, and run a linter yourself, you decide to use an action from the GitHub Actions Marketplace that does what you need. You give it a test run, and it works! Since it does what you want, you set up a workflow in your repo that uses it:
> 
```
 - name: Lint code
 uses: someperson/lint-action@v1
```
> 
> After months of using this action, you suddenly start having issues with your API keys stolen and abused. After some investigating, you find out that the author of the third party linter action recently pushed an update to the GitHub Marketplace. You go to the source repo for the action and see that code was recently added to the linter action to send environment variables to some random web address.
> 
> In that hypothetical scenario, the author of the third party action (or someone who hijacked their account) added malicious code to the action and re-tagged it as “v1”. Everyone using “someperson/linter-action@v1” was now running the malicious code in their workflows. Now that we see the threat from this scenario, how do we protect ourselves from it?

[
GitHub Actions Security Best Practices](https://blog.gitguardian.com/github-actions-security-cheat-sheet/)